### ISSUE TITLE
Überschneidungen Backend

### ISSUE THREAD

**Issue by jwssnr** [jwssnr/feeef2#7]
Wie weit soll die Ausbildung des Frontend-Entwicklers ins Backend greifen, bzw. in welchen Inhalten sollen Applikationsentwickler und Frontender Schnittmengen haben? Sind die Begriffe Backend und Frontend überhaupt noch zeitgemäss? Immer mehr «einstige» Aufgaben des Backends haben sich ins Frontend verlagert. Somit würde der Frontend-Entwickler immer mehr zum Full-Stack-Entwickler, was aber nicht die Idee ist. Es braucht nachwievor die drei groben Disziplinen Design, Design-Realisation, und Applikation «im Hintergrund», wobei eben Hintergrund nicht mehr unbedingt Backend ist.

**Comment by signalwerk** [comment]
@jwssnr du sprichst ein wichtiges Thema an. Damit verknüpft ist natürlich auch der name «Frontend-Entwickler EFZ». Ich finde den Titel gut, da er die technischen Skills sehr präzise beschreibt. Der Bereich Backend/Design kommt hier jedoch einwenig kurz. 
Grundsätzlich glaube ich, dass ein FE EFZ in der Lage sein sollte ohne eine weitere Backend-Person eine einfache Website an den Start zu bringen (mit CMS). Für komplexere Backends oder Sites mit grosser Last braucht es dann externe. Dank serverless und Decoupled CMS ist das Backend heute auch in einem anderen Kontext zu sehen.

**Comment by jwssnr** [comment]
Genau. Bin ebenfalls der Meinung, Frontender müssen ein verbreitetes CMS aufsetzen und dafür Templates schreiben können. Zu den Begriffen nochmals ein paar Gedanken hier im Logbuch: https://frontend-entwickler.ch/frontend-vs-backend/ (sollte bei Gelegenheit noch die Single-Pages ausarbeiten ;-)

**Comment by jwssnr** [comment]
Hab zum reinen Thema Berufsbezeichnung ein zusaätzliches Issue eröffnet: #10
